### DISCUSSION TITLE
security while database running unlocked in background?

### DISCUSSION THREAD

**Discussion by cotangente** [keepassxreboot/keepassxc#6773] [Q&A] [answered]
hello,

I'm running debian and I was wondering if there is any known security risk if I have my database running unlocked in my working hours all day long.

I know the basics that one process can not read other process memory, but, if I have zoom or teams installed, can it read somehow my database?

**Comment by phoerious** [comment, marked as answer]
Not without a security vulnerability in your operating system. I would say the risk is minimal. A much larger risk is probably that someone accesses your PC while you are away or that you catch a spyware that records your keypresses and screen content.
